### PR TITLE
Updates

### LINKED ISSUE #15: Rename `attr` to `attrs`
It's kinda nit-picky, but I think that "attrs" is a bit more clear in indicating that there may be any number of attributes.

### PR DESCRIPTION
**Pull request by nwlandry**
This PR does the following:
* Fixes #15.
* Moves extraneous files related to performance testing, integration testing, and other functionality to `_OBS`
* Moved all of the files except for the schema and the CHANGELOG out of the `schemas` folder.
* Made two requirements files: one for unit tests and one for the tutorials.
* Updated the README.
* Moved all of the Jupyter notebooks to the `tutorials` folder. Made a notebook for each library and an "integration" notebook showcasing the ability to exchange network datasets between libraries.

### CODE REVIEW

**Comment by nwlandry** [comment]
@colltoaction --- are you able to review this?

File: README.md [hunk 1]
```diff
@@ -47,8 +51,9 @@ The only option is `--silent`, which suppresses all detailed warnings. Regardles
 ```python
 import fastjsonschema
 import json
+import requests
 
-schema = json.load(open("hif_schema_v0.1.0.json","r"))
+schema = requests.get("https://raw.githubusercontent.com/pszufe/HIF_validators/main/schemas/hif_schema_v0.1.0.json").json()
```
Comment (colltoaction) [inline README.md:56]: I think we need to keep the local link so that GH Actions can test the schema changes

**Review by colltoaction — COMMENTED** [review]
The rest looks good!

File: README.md (hunk 1 above)
Comment (nwlandry) [reply README.md:56]: I don't thing that GitHub actions is testing the README so my thought was to have a code snippet that users can run without cloning the repo.

Comment (colltoaction) [reply README.md:56]: My bad haha. LGTM.

**Comment by nwlandry** [comment]
Thanks so much for the review, @colltoaction! Really appreciate it!

**Pull request merged**